zmon_aws_agent/aws.py

```diff
@@ -566,6 +566,56 @@ def get_rds_instances(region, acc):
     return rds_instances
 
 
+def get_certificates(region, acc):
+    iam_client = boto3.client('iam', region_name=region)
+    acm_client = boto3.client('acm', region_name=region)
+
+    entities = []
+
+    try:
+        server_certs = iam_client.list_server_certificates()['ServerCertificateMetadataList']
+
+        acm_certs = acm_client.list_certificates()['CertificateSummaryList']
+
+        for cert in server_certs:
+            e = {
+                'id': entity_id('cert-iam-{}[{}:{}]'.format(cert['ServerCertificateName'], acc, region)),
+                'type': 'certificate',
+                'infrastructure_account': acc,
+                'region': region,
+                'created_by': 'agent',
+                'certificate_type': 'iam',
+                'name': cert['ServerCertificateName'],
+                'arn': cert['Arn'],
+                'status': 'ISSUED',
+                'expiration': datetime.isoformat(cert['Expiration']),
```

Comment:
Actually it works (tests cover this part actually), but in a hackish way that I overlooked. The way it is used now: ``isoformat()`` gets a datetime object in the ``self`` arg, so it works 😄 

Will modify it to call isoformat on the datetime object itself.